docling_jobkit/datamodel/chunking.py

```diff
@@ -0,0 +1,112 @@
+import enum
+from typing import Annotated, Literal, Optional
+
+from pydantic import BaseModel, Field
+
+from docling.datamodel.base_models import ConversionStatus, ErrorItem
+from docling.utils.profiling import ProfilingItem
+
+
+class ChunkerType(str, enum.Enum):
+    """Choice of the chunkers available in Docling."""
+
+    HIERARCHICAL = "hierarchical"
+    HYBRID = "hybrid"
+
+
+class BaseChunkerOptions(BaseModel):
+    """Configuration options for document chunking using Docling chunkers."""
+
+    chunker: ChunkerType
+
+    use_markdown_tables: Annotated[
+        bool,
+        Field(
+            description="Use markdown table format instead of triplets for table serialization.",
+        ),
+    ] = False
+
+    include_raw_text: Annotated[
+        bool,
+        Field(
+            description="Include both raw_text and text (contextualized) in response. If False, only text is included.",
+        ),
+    ] = True
+
+
+class HierarchicalChunkerOptions(BaseChunkerOptions):
+    """Configuration options for the HierarchicalChunker."""
+
+    chunker: Literal[ChunkerType.HIERARCHICAL] = ChunkerType.HIERARCHICAL
+
+
+class HybridChunkerOptions(BaseChunkerOptions):
+    """Configuration options for the HybridChunker."""
+
+    chunker: Literal[ChunkerType.HYBRID] = ChunkerType.HYBRID
+
+    max_tokens: Annotated[
+        int,
+        Field(
+            description="Maximum number of tokens per chunk. Default is 512.",
+            gt=0,
+            le=32768,  # Reasonable upper limit
+        ),
+    ] = 512
+
+    tokenizer: Annotated[
+        Optional[str],
+        Field(
+            description="HuggingFace model name for custom tokenization. If not specified, uses 'sentence-transformers/all-MiniLM-L6-v2' as default.",
+            examples=[
+                "Qwen/Qwen3-Embedding-0.6B",
+                "sentence-transformers/all-MiniLM-L6-v2",
+            ],
+        ),
+    ] = None
```

Comment:
I would the determine the fallbacks of these two on the same level (e.g. by dropping the `default_tokenizer` from the `DocumentChunkerConfig` and bringing the default value here).